lib/RulesDeploy.js

```diff
@@ -93,21 +93,38 @@ RulesDeploy.prototype = {
     var self = this;
     return gcp.rules.testRuleset(self.options.project, files).then(function(response) {
       if (response.body && response.body.issues && response.body.issues.length > 0) {
-        var add = response.body.issues.length === 1 ? "" : "s";
-        var message = "Compilation error" + add + " in " + chalk.bold(filename) + ":\n";
+        var warnings = [];
+        var errors = [];
         response.body.issues.forEach(function(issue) {
-          message +=
-            "\n[" +
+          var issueMessage =
+            "[" +
             issue.severity.substring(0, 1) +
             "] " +
             issue.sourcePosition.line +
             ":" +
             issue.sourcePosition.column +
             " - " +
             issue.description;
+
+          if (issue.severity === "ERROR") {
+            errors.push(issueMessage);
+          } else {
+            warnings.push(issueMessage);
+          }
         });
 
-        return utils.reject(message, { exit: 1 });
+        if (warnings.length > 0) {
+          warnings.forEach(function(warning) {
+            utils.logWarning(warning);
```

Comment:
I worry that these will not clearly be parseable as rules compilation warnings if someone is doing a deploy of everything. Maybe something like:

```js
utils.logWarning("Rules compilation warnings in " + chalk.bold(filename) + ":\n  " + warnings.join("\n  "))
```

would be more parseable?